templates/LDA_Preprocessing.ipynb

```diff
@@ -1,15 +1,941 @@
 {
```

Comment:
<p>The and/or is confusing here. Don't we need to pick either clusters or markers for spatial LDA to operate on?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/468/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>You must specify one or the other at the featurization step.  If you force people to specify one or the other here then they would have to make two separate formatted cell tables instead of just the one.</p>

Reply:
Got it. I think explaining that here would be useful. Tell people that they'll be able to generate a spatial model based either on clusters or markers, but to support both options they can specify that here.

Reply:
Sounds good